template.tpl

```diff
@@ -38,19 +38,43 @@ ___TEMPLATE_PARAMETERS___
     "name": "lf_tracker_id",
     "displayName": "Leadfeeder Tracker ID",
     "simpleValueType": true,
-    "help": "You can find your Leadfeeder TrackerID in \u003ca href\u003d\"https://app.leadfeeder.com/l/settings/company/tracking\"\u003eLeadfeeder Tracker Settings\u003c/a\u003e"
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
+    "help": "You can find your Leadfeeder TrackerID in \u003ca href\u003d\"https://app.leadfeeder.com/l/settings/company/tracking/script\" target\u003d\"_blank\"\u003eLeadfeeder Tracker Settings\u003c/a\u003e",
+    "alwaysInSummary": true
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "autotrack",
+    "checkboxText": "Automatic tracking",
+    "simpleValueType": true,
+    "defaultValue": true,
+    "help": "When this checkbox is enabled the tracker will send a pageview event anytime the tag is included.\nDisabling this field might be useful e.g. when Leadfeeder tracker is used with Single Page Applications.\nYou can read more about SPA tracking \u003ca href\u003d\"https://help.leadfeeder.com/en/articles/4519005-customize-your-site-tracking-with-leadfeeder-javascript-api\" target\u003d\"_blank\"\u003eLeadfeeder Help Center\u003c/a\u003e",
+    "alwaysInSummary": true
   }
 ]
 
 
 ___SANDBOXED_JS_FOR_WEB_TEMPLATE___
 
-const setInWindow = require('setInWindow');
 const injectScript = require('injectScript');
+const createArgumentsQueue = require('createArgumentsQueue');
+const encodeUriComponent = require('encodeUriComponent');
+
+// Handle both the case when someone provides a raw tracker ID and the case when the tracker ID is prepended with 'v1_'.
+const versionlessTrackerId = data.lf_tracker_id.indexOf('v1_') == 0 ? data.lf_tracker_id.split('_')[1] : data.lf_tracker_id;
 
-setInWindow('ldfdr', {}, false);
+// The following line creates a `ldfdr` function in `window`.
+// Once called the function appends arguments to `ldfdr._q` array.
+// Leadfeeder Tracker uses the arguments queue during initialization.
+const ldfdr = createArgumentsQueue('ldfdr', 'ldfdr._q');
```

Comment:
This single line does exactly what our "standard" snippet does:
```javascript
window.ldfdr=window.ldfdr||function(){(ldfdr._q=ldfdr._q||[]).push([].slice.call(arguments));};
```

First I wanted to simply store this function in the window like this:
```javascript
setInWindow('ldfdr', function(){(ldfdr._q=ldfdr._q||[]).push([].slice.call(arguments));}, false);
```

but unfortunately it doesn't work because `ldfdr` isn't available when this funciton is called. I tried all possible variations and [`createArgumentsQueue`](https://developers.google.com/tag-manager/templates/api#createargumentsqueue) simply seems to be the best solution.